entity-service-impl/src/main/java/org/hypertrace/entity/query/service/EntityAttributeMapping.java

```diff
@@ -16,9 +18,13 @@
   private static final String ATTRIBUTE_SERVICE_HOST = "attribute.service.config.host";
   private static final String ATTRIBUTE_SERVICE_PORT = "attribute.service.config.port";
   static final String ENTITY_ATTRIBUTE_DOC_PREFIX = "attributes.";
+  public static final String TYPE_STRING_ARRAY = "TYPE_STRING_ARRAY";
+  public static final String VALUE_KIND = "valueKind";
+  public static final String SUB_DOC_PATH = "subDocPath";
 
   private final CachingAttributeClient attributeClient;
   private final Map<String, String> explicitDocStoreMappingsByAttributeId;
+  private final List<String> multiValueAttributes;
```

Comment:
I think we call it `multiValuedAttributes` at some place, and `multiValueAttributes` at other places. Can we have a consistent naming scheme? Either is fine